Velaptor/Graphics/Renderers/ITextureRenderer.cs

```diff
@@ -54,6 +54,48 @@ public interface ITextureRenderer
     /// </remarks>
     void Render(ITexture texture, int x, int y, int layer = 0);
 
+    /// <summary>
+    /// Renders the given texture at the given <paramref name="x"/> and <paramref name="y"/> coordinates.
```

Comment:
Update this to mention the angle

```suggestion
    /// Renders the given texture at the given <paramref name="x"/> and <paramref name="y"/> coordinates and the given <paramref name="angle"/>.
```